.github/mergify.yml

```diff
@@ -1,4 +1,73 @@
 pull_request_rules:
+- name: auto-merge
+  description: Automatically merge to main with >= 2 approvals. All requested reviews have given feedback, no changes requested, and CI has passed.
+  conditions:
+    - "#approved-reviews-by>=2"
+    - "#review-requested=0"
+    - "#changes-requested-reviews-by=0"
+    - label!=hold
+    - label!=do-not-merge
+    - label!=needs-rebase
+    - check-success=DCO
+
+    - or:
+      # base branch is main or a release branch
+      - base=main
+      - base ~=^release
+
+    # The files conditions regex should match the globs in workflow files
+    # If workflow configuration files in .github/ are changed, the actionlint check must pass
+    - or:
+      
+      - and:
+        - check-success=actionlint
```

Comment:
This will be a problem on a `release-` branches since the actionlint workflow is not configured for those branches. 

https://github.com/instructlab/sdg/blob/ec9545b137db372459bf1741188098843427b228/.github/workflows/actionlint.yml#L6-L7

You should update that workflow to include release branches.

https://github.com/instructlab/instructlab/blob/af78b8c144173d65ca1cc0d546d4ea867ef3d4ac/.github/workflows/actionlint.yml#L8